src/promptflow-tools/tests/utils.py

```diff
@@ -11,6 +11,29 @@ def __getattr__(self, item):
         return super().__getattribute__(item)
 
 
+class Deployment:
+    def __init__(self, name, model_name, version):
+        self.name = name
+        self.properties = Properties(model_name, version)
+
+
+class CustomException(Exception):
```

Comment:
What is the class used for? Just for test?

Reply:
Just for test